.env.dist

```diff
@@ -1,27 +1,20 @@
 # App configuration
-ROBOT_EMAIL_ADDRESS=noemail@example.com
-ROBOT_EMAIL_NAME=Robot
+SENDER_ADDRESS=noemail@dev.caesar.team
```

Comment:
Please do not use real data in dist files.

Reply:
replaced